app/views/tag/_subscribe_button.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if current_user %>
```

Comment:
It will be great if you can add comments to explain the functionality aka documentation.